ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/dialog-directives.ts

```diff
@@ -7,21 +8,25 @@ import {
 } from '@angular/material/dialog';
 
 @Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
   selector: '[fudis-dialog-title]',
 })
-export class FudisDialogTitle extends MatDialogTitle {}
+export class FudisDialogTitleDirective extends MatDialogTitle {}
 
 @Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
   selector: 'fudis-dialog-content',
 })
-export class FudisDialogContent extends MatDialogContent {}
+export class FudisDialogContentDirective extends MatDialogContent {}
 
 @Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
   selector: 'fudis-dialog-actions',
 })
-export class FudisDialogActions extends MatDialogActions {}
+export class FudisDialogActionsDirective extends MatDialogActions {}
 
 @Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
   selector: '[fudis-dialog-close]',
 })
-export class FudisDialogClose extends MatDialogClose {}
+export class FudisDialogCloseDirective extends MatDialogClose {}
```

Comment:
Tätä sääntöä ehkä joutuu vähän säätämään jos näitä alkaa tulemaan lisää. Angular styleguide haluais että käytetään cammelCasea, mutta nää angular-materiaalista tulevat direktiivit on ehkä vähän erikoistapauksia.